src/loaders/ImageLoader.js

```diff
@@ -46,32 +46,44 @@ Object.assign( ImageLoader.prototype, {
 
 		var image = document.createElementNS( 'http://www.w3.org/1999/xhtml', 'img' );
 
-		image.addEventListener( 'load', function () {
+		var loadListener, errorListener;
+
+		loadListener = function () {
```

Comment:
`function onLoad() {` maybe?

Reply:
Was thinking the same initially, but there is a user-supplied `onLoad` callback passed into the function already. Open to other ideas tho.

Reply:
Went ahead and renamed to `onInternalLoad`, `onInternalError`, also changed the declaration syntax to match your suggestion

Reply:
Thanks!